packages/core/src/interfaces/charts.ts

```diff
@@ -509,11 +508,11 @@ export interface TreeChartOptions extends BaseChartOptions {
  */
 export interface CirclePackChartOptions extends BaseChartOptions {
 	circlePack?: {
-		circles: {
+		circles?: {
```

Comment:
Most of the circle pack chart demos did not have this property so making it optional